services/kaze/controllers/application.go

```diff
@@ -242,15 +245,23 @@ func FetchMetrics(c *gin.Context) {
 		}
 	}
 
-	metrics := mongo.FetchContainerMetrics(types.M{
-		mongo.NameKey: appName,
-		mongo.TimestampKey: types.M{
-			"$gte": time.Now().Unix() - timeSpan,
-		},
-	})
-
-	c.JSON(200, gin.H{
-		"success": true,
-		"data":    metrics,
+	chanStream := make(chan bool)
+	go func() {
+		defer close(chanStream)
+		chanStream <- true
+		time.Sleep(2 * configs.ServiceConfig.Mizu.MetricsInterval)
```

Comment:
the `2 *` part is not required

Reply:
2 * part was due to you reply to the question I asked in slack for time interval between two Server Sent events

Reply:
My bad, it should be just `configs.ServiceConfig.Mizu.MetricsInterval`